{"name": "test_3", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 6\n  +2\n  -6", "trace": "setup_module = <Pytest.test_allure2.TestClass object at 0x0000020170ED6970>\n\n    @allure.story(\"功能测试用例3\")\n    def test_3(setup_module):\n        print('Test_3 called.')\n>       assert 2 == 1 + 5\nE       assert 2 == 6\nE         +2\nE         -6\n\ntest_allure2.py:41: AssertionError"}, "start": 1643180640524, "stop": 1643180640524, "uuid": "7d9a7d6e-378a-415e-a63a-ce0c3f03c3ae", "historyId": "3cd3021efd2c9cdca6e71c38112d6732", "testCaseId": "c98b9fe8fa316bbd69ea874cb4139972", "fullName": "test_allure2.TestClass#test_3", "labels": [{"name": "story", "value": "功能测试用例3"}, {"name": "feature", "value": "测试用例功能"}, {"name": "suite", "value": "test_allure2"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "DESKTOP-V45JC51"}, {"name": "thread", "value": "14500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure2"}]}